IDL/key.idl

```diff
@@ -0,0 +1,98 @@
+struct KeyedShortStruct
+{
+    @Key short key_short;
```

Comment:
`@Key` is eProsima alias for the IDL standard `@key`. Maybe we can use in some structures `@key` instead of the non-standard form of `@Key`

Reply:
Made a few @key in https://github.com/eProsima/dds-types-test/pull/6/commits/57ec2286866c7628f3819fc1e9a58f970feabce6